lametro/templates/event/_related_bills.html

```diff
@@ -14,7 +14,7 @@ <h4>Board Reports</h4>
       {% with associated_bill=report.related_entities.all.0.bill %}
       <tr>
         <td><strong>{{ report.notes.0 | parse_agenda_item }}</strong></td>
-        <td>{{associated_bill.identifier}} {{report.description | short_blurb}} {{ associated_bill.inferred_status | inferred_status_label | safe }}</td>
+        <td class="blurb">{{associated_bill.identifier}} {{report.description | short_blurb}} {{ associated_bill.inferred_status | inferred_status_label | safe }}</td>
```

Comment:
How about adding this class to the parent table, so all of its contents are a bit larger?